notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>please, add version of gradio, for example last version: gradio==4.0.2</p><p>I had version 3.18 and it doesn't update this way, but version 3.18 doesn't have a 'select' method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
fixed